src/main/java/com/labsynch/labseer/chemclasses/jchem/ChemStructureServiceJChemImpl.java

```diff
@@ -926,8 +905,8 @@ else if (plainTable != null && inputCdIdHitList == null && searchType.equalsIgno
 
 	@Override
 	@Transactional
-	public CmpdRegMolecule[] searchMols(String molfile, String structureTable, int[] inputCdIdHitList, 
-			String plainTable, String searchType, Float simlarityPercent, int maxResults) {
+	public CmpdRegMolecule[] searchMols(String molfile, StructureType structureType, int[] inputCdIdHitList, 
+			SeasrchType searchType, Float simlarityPercent, int maxResults) {
```

Comment:
typo in `SeasrchType` here. Please test a compile with JChem libs and one with Indigo just to make sure they do compile.

Reply:
Thanks, yeah. Just fixed a bunch of issues with both Indigo and Jchem builds

Reply:
Fixed.